### PR TITLE
Removed appUrl authentication stripping behavior

### LINKED ISSUE #29: Unable to use local auth stubs
There is an issue with how the login window constructs it’s URLs – it ignores ports in URLs. This is not going to be an issue when it’s hosted, I don’t think Antares will ever run on anything but standard ports, but for my local auth stub, it only works when running on port 80. The issue lies here.

### PR DESCRIPTION
**Pull request by mamaso**
resolves #29 @danderson00

### CODE REVIEW

**Comment by mamaso** [comment]
Any ideas why this stripping behavior was in place?

**Comment by danderson00** [comment]
No idea. If E2E tests pass, lets just remove it altogether.

File: sdk/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/authentication/LoginManager.java [hunk 1]
```diff
@@ -150,7 +150,7 @@ public LoginManager(MobileServiceClient client) {
             loginAsyncDoneUriFragment = UriHelper.CombinePath(this.mClient.getLoginUriPrefix(), "done");
         }
 
-        URL appUrl = UriHelper.createHostOnlyUrl(mClient.getAppUrl());
+        URL appUrl = mClient.getAppUrl();
```
Comment (phvannor) [inline sdk/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/authentication/LoginManager.java]: Yes, the '.auth/login' need to go onto the host, not the URL used by the client.  

Esp important if you need to do a SxS install for teh next major version.  I.e. I can run my original site on phil-net.azurewebsites.net and setup my new one on phil-net.azurewebsites.net/v2.  Then my new clients are initialized with the phil-net/v2 URL.  But EA should ideally go to /.auth/login not v2/.auth/login.

Do NOT remove this logic.

Comment (danderson00) [inline sdk/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/authentication/LoginManager.java]: No worries. We just need to add the port in the function below

Comment (mamaso) [inline sdk/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/authentication/LoginManager.java]: Awesome, thanks for the clarification Phil

**Comment by danderson00** [comment]
Awesome, I'll get you to give me a hand with building and testing the client. Cheers!

File: sdk/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/authentication/UriHelper.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public static String normalizeParameters(HashMap<String, String> parameters) {
     */
     public static URL createHostOnlyUrl(URL appUrl) {
         try {
-            appUrl = new URL(appUrl.getProtocol(), appUrl.getHost(), "/");
+            appUrl = new URL(appUrl.getProtocol(), appUrl.getHost(), appUrl.getPort(), "/");
```
Comment (soninaren) [inline sdk/src/sdk/src/main/java/com/microsoft/windowsazure/mobileservices/authentication/UriHelper.java]: Can you add a unit test to verify port is properly added.

**Comment by soninaren** [comment]
Merge conflict sending another PR

**Pull request closed without merging**